Makefile

```diff
@@ -102,11 +102,14 @@ uninstall:
 
 clean:
 	$(RM) -r out/Makefile $(NODE_EXE) $(NODE_G_EXE) out/$(BUILDTYPE)/$(NODE_EXE) \
-                out/$(BUILDTYPE)/node.exp
+		out/$(BUILDTYPE)/node.exp
 	@if [ -d out ]; then find out/ -name '*.o' -o -name '*.a' -o -name '*.d' | xargs $(RM) -r; fi
 	$(RM) -r node_modules
 	@if [ -d deps/icu ]; then echo deleting deps/icu; $(RM) -r deps/icu; fi
 	$(RM) test.tap
+	$(RM) -r test/tmp*
```

Comment:
I'm not sure we need this, if we never had it before, then why not just stick with the new one? (partly a question for @Trott )

If we are adding it (and it's already legacy) then could you add a comment saying `# Remove this at some point`